### PR TITLE
Resolution relative paths and correction flag typos

### PR DESCRIPTION
**Pull request by cavelandiah**

### CODE REVIEW

File: script/miRNAture [hunk 1]
```diff
@@ -71,29 +71,38 @@ GetOptions (
...
+if ($data_path !~ /^\/\.+\//){
+	$data_path = abs_path($data_path);
 }
 
 my $config_file = Mir::ConfigurationFile->new (
 	stage => $stagesRun,
-	specie_name => $name_specie,
-	specie_tag => $specie_tag,
-	specie_genome => $specie_genome_path,
+	specie_name => $name_species,
```
Comment (jfallmann) [inline script/miRNAture:103]: Would make sense to replace all specie_ with species_
also in other modules for consistency

**Review by jfallmann — CHANGES_REQUESTED** [review]
Can you replace specie with species in all modules?

File: script/miRNAture (hunk 1 above)
Comment (cavelandiah) [reply script/miRNAture]: Changed on flags and modules.

**Review by jfallmann — APPROVED** [review]
Seems ok

**Pull request merged**